mopidy_spotify/translator.py

```diff
@@ -7,7 +7,7 @@
 logger = logging.getLogger(__name__)
 
 
-class memoized:
+class Memoized:
```

Comment:
I'd rather `# noqa` this and keep the decorator lowercase. Never seen an uppercase decorator before, so that felt very strange.